package.json

```
@@ -9,7 +9,7 @@
     "lint:eslint": "eslint --ext .ts src --max-warnings 0",
     "lint:tsc": "tsc",
     "prepare": "husky install",
-    "prepublishOnly": "npm run build",
+    "prepublishOnly": "npm i && npm run build",
```

Any time we run `npm i` we need to run the test suite to make sure the new versions of npm packages didn't break anything. We can either do that or run `npm ci`, as in the suggestion.
```suggestion
    "prepublishOnly": "npm ci && npm run build",
```
We can also decide to take the risk, that's also a valid choice. In that case you can resolve this without action.